src/workflows/airqo_etl_utils/airqo_utils.py

```
@@ -581,17 +580,9 @@ def calibrate_data(data: pd.DataFrame) -> pd.DataFrame:
-        # Load sites from cache
-        sites = DataUtils.load_cached_data(local_file_path, MetaDataType.SITES.str)
+        sites = DataUtils.get_sites()
         if sites.empty:
-            try:
-                sites = pd.DataFrame(AirQoApi().get_sites())
-            except Exception as e:
-                logger.exception(
-                    "An error has occured while fetching devices from the api"
-                )
-                raise RuntimeError("Failed to fetch sites data from the API") from e
+            raise RuntimeError("Failed to fetch sites data from the cache/API") from e
```

Ruff (0.8.2)

Undefined name `e`

(F821)